static/src/javascripts/projects/common/views/ui/notifications-follow-link.html

```diff
@@ -0,0 +1,7 @@
+ <button class="button button--large button--tertiary js-notifications__button notifications__button notifications-follow-input--solo notifications__button--<%= subscribed ? 'subscribed' : 'unsubscribed' %>">
+    <%= icon %>
+    <span class="live-notifications__label live-notifications__label--unsubscribed">Get alerts for this story</span>
+    <span class="live-notifications__label live-notifications__label--subscribed">Alerts on this story</span>
```

Comment:
Instead of using CSS, you could pass in some state to the template to decide which label to show:

``` html
<% if (isSubscribed) { %>
    <span class="live-notifications__label live-notifications__label--subscribed">Alerts on this story</span>
<% } else { %>
    <span class="live-notifications__label live-notifications__label--unsubscribed">Get alerts for this story</span>
<% } %>
```